src/deployable_example.cpp

```diff
@@ -170,7 +170,7 @@ int start_hr = 8;
 int end_hr = 17;
 
 bool disable_deep_sleep_for_notifications = false;
-bool disable_deep_sleep_until_reset = false;
+bool disable_deep_sleep_until_reset = true;
```

Comment:
Why do we want to disable deep sleep on the first loop?

Reply:
So that the access point remains active, and we can still access the local web page to update configuration settings later on.